keylime/tpm/tpm_main.py

```diff
@@ -29,8 +29,9 @@
 logger = keylime_logging.init_logging("tpm")
 
 
-class tpm(tpm_abstract.AbstractTPM):
+class tpm:
```

Comment:
Maybe a chance to rename it as "Tpm". Usually lowercase names for classes are for primitive types (list, dict) but not for user defined classes